app/views/layouts/application.html.slim

```diff
@@ -1,6 +1,8 @@
 doctype html
 html
   head
+    = render 'layouts/shared/google_analytics'
```

Comment:
keep JS scripts with JS scripts within head

Reply:
it's ok here, but just change to `render_google_analytics` how Roman suggested